rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -469,38 +470,30 @@ def _extract_features(
             sparse_sequence_features = None
             dense_sequence_features = None
 
-        return (
-            sparse_sequence_features,
-            sparse_sentence_features,
-            dense_sequence_features,
-            dense_sentence_features,
-        )
+        out = {}
```

Comment:
does it makes sense to make `out` dict of dict as well

Reply:
yes, that is on my list, but as it is a bit more complicated I postponed it for now